src/NodaTime.Demo/PeriodDemo.cs

```diff
@@ -177,5 +177,28 @@ public void Between_LocalDateTimesWithGivenUnits()
             Assert.AreEqual(23, period.Hours);
             Assert.AreEqual("P2Y264DT23H", period.ToString());
         }
+
+        [Test]
+        public void Normalize()
+        {
+            var original = new PeriodBuilder { Weeks = 2, Days = 5 }.Build();
+
+            var actual = Snippet.For(original.Normalize());
+
+            Assert.AreEqual(
+                new PeriodBuilder { Days = 19 }.Build(), 
+                actual);
+        }
+
+        [Test]
+        public void ToDuration()
+        {
+            Period oneDayAsPeriod = Period.FromDays(1);
+
+            var actual = Snippet.For(oneDayAsPeriod.ToDuration());
+
+            Assert.IsFalse(oneDayAsPeriod.HasTimeComponent);
```

Comment:
Could you make this Assert.False instead of Assert.IsFalse? It looks like I have special handling for the former, but not the latter. I can add IsFalse/IsTrue handling later if we want.

Reply:
Fixed.